runtime/lua/vim/ui/img/utils.lua

```diff
@@ -0,0 +1,266 @@
+local M = {}
+
+---@alias vim.ui.img.utils.Unit 'cell'|'pixel'
+
+---@class vim.ui.img.utils.Codes
+M.codes = {
+  ---Hides the cursor from being shown in terminal.
+  CURSOR_HIDE            = '\027[?25l',
+  ---Restore cursor position based on last save.
+  CURSOR_RESTORE         = '\0278',
+  ---Save cursor position to be restored later.
+  CURSOR_SAVE            = '\0277',
+  ---Shows the cursor if it was hidden in terminal.
+  CURSOR_SHOW            = '\027[?25h',
+  ---Queries the terminal for its background color.
+  QUERY_BACKGROUND_COLOR = '\027]11;?',
+  ---Disables scrolling mode for sixel.
+  SIXEL_SCROLL_DISABLE   = '\027[?80l',
```

Comment:
FYI, this is probably not doing what you think it's doing. Mode `?80` is Sixel Display Mode, which is the opposite of Sixel scrolling. By disabling this mode you're actually _enabling_ scrolling. You're doing the right thing, though, because if Sixel Display Mode was enabled, all images would be drawn in the top left corner. It's just that the comment is incorrect.

Reply:
Ah, so I'm really disabling sixel display mode, which in turn is also disabling sixel scrolling?

Was following the bottom of this page: https://vt100.net/docs/vt3xx-gp/chapter14.html

So, really, this should be named something like `SIXEL_DISPLAY_MODE_DISABLE`?

Reply:
> Ah, so I'm really disabling sixel display mode, which in turn is also disabling sixel scrolling?

No, you're disabling sixel display mode, which in turn is _ENABLING_ sixel scrolling. They're the exact opposite of each other.

The documentation in the VT340 manual is unfortunately wrong, and has been a source of much confusion. So for many years, at least half of the Sixel terminals had the mode implemented backwards. I'd usually advocate for leaving it unchanged, because it has the potential to make your app unusable on some terminals, but that's probably less of a risk these days.

> So, really, this should be named something like `SIXEL_DISPLAY_MODE_DISABLE`?

That would be more accurate, yes.